### PR TITLE
add spanish

### PR DESCRIPTION
**Pull request by alvarosamudio**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (XMuli) [inline .vscode/settings.json]: It looks like this is a useless file  （.vscode/settings.json）

File: docs/01.DTK/01.Dtk文档总集合.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Dtk文档总集合
-date: 2020-12-11 10:56:23
+date: 2020-12-17 10:56:23
```
Comment (XMuli) [inline docs/01.DTK/01.Dtk文档总集合.md]: date: 2020-12-11 10:56:23
update: 2020-12-17 10:56:23

Maybe you are trying to express this?

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ module.exports = {
                 items: [
                     { text: '简体中文', link: '/' },
                     { text: 'English', link: '/pages/ebde83/' },
+                    { text: 'Español', link: '/pages/722460/' },
```
Comment (XMuli) [inline docs/.vuepress/config.js:21]: nice

**Review by XMuli — CHANGES_REQUESTED** [review]
Your commits are a bit confusing to me, it looks like there are a lot of accidental commits mixed in?

For example, the files under `docs/01.DTK/xxx` and `docs/en-US/xxx` do not need to be modified; but `docs/es/xxx` is perfectly fine

To translate to another language, just add a new folder, like your `es`, and add changes to the files in it, while the contents of the other files don't have to be changed.

Of course `docs/docs/.vuepress/config.js` is required

**Comment by XMuli** [comment]
After modification, this submission will be merged

File: .vscode/settings.json (hunk 1 above)
Comment (alvarosamudio) [reply .vscode/settings.json]: the file was lost in the commit is part of the configuration of a plugin for visual studio code, to help me with the translation

File: docs/01.DTK/01.Dtk文档总集合.md (hunk 2 above)
Comment (alvarosamudio) [reply docs/01.DTK/01.Dtk文档总集合.md]: I don't know why the date was changed

**Comment by alvarosamudio** [comment]
I wanted to add to the scheme of the Spanish files which is the same as the English translation files, that's what gave conflict, but I fixed it

**Review by XMuli — CHANGES_REQUESTED** [review]
**These three are also missing (and need to be delete)**

docs/04.关于/01.如何投稿.md
docs/en-US/03.Community creations/02.Deepin Custom Modify Window Corner Rounding Size.md
docs/en-US/04.About/01.How to do internationalization.md

**Review by XMuli — APPROVED** [review]
nice

**Pull request merged**